### PR TITLE
chore(teams): refactor teams page

### PR DESCRIPTION
**Pull request by mtrezza**
- Generally refactored teams page for better readability
- Simplified tables for easier maintainability
- Added permissions review security policy
- Added removal policy
- Removed PCM members from contributors list
- Simplified member info
- Fixed GH image URL (no resizing for auto-gen. avatars)
- Added TOC

TODO:
- [x] ~~Add missing team members~~ Current member list is available on org page.
- [x] ~~Add "Joined" dates where known~~ Unnecessary

### CODE REVIEW

**Comment by mtrezza** [comment]
@davimacedo I refactored the teams page, not only was it completely outdated but the layout was confusing.

We are using the `img` tags now, because it seems that the original GH image URL does not respect resizing for profile pictures that are just avatars. So if you add a new member, you can just replace the `/u/xxxx/` in the `img` tag for the new member UID instead of replacing the whole img URL.

File: TEAM.md [hunk 1]
```diff
@@ -1,51 +1,49 @@
-# Team
+# Team <!-- omit in toc -->
 
-Below are the current members of the Parse Community. Please see our [Governance](GOVERNANCE.md) document for details on the meanings of the roles listed below.
```
Comment (davimacedo) [inline TEAM.md:3]: I think this link does not exist. It should point out to README.md

File: TEAM.md [hunk 2]
```diff
@@ -1,51 +1,49 @@
...
+# Project Management Committee (PMC)
 
-|         | ![John Carlson](https://avatars1.githubusercontent.com/u/1459320?s=180&v=4) | ![Roger Hu](https://avatars1.githubusercontent.com/u/326857?s=180&v=4) | ![Tobias](https://avatars3.githubusercontent.com/u/5549565?s=180&v=4) | ![Omair Vaiyani](https://avatars2.githubusercontent.com/u/5912209?s=180&v=4) | ![Antoine Cormouls](https://avatars3.githubusercontent.com/u/27959372?s=180&v=4) |
-|---------|:---:|:---:|:---:|:---:|:---:|
-| Name:   | [__John Carlson__](https://github.com/Jawnnypoo) | [__Roger Hu__](https://github.com/rogerhu) | [__Tobias__](https://github.com/TobiasPott) | [__Omair Vaiyani__](https://github.com/omairvaiyani) | [__Antoine Cormouls__](https://github.com/Moumouls) |
-| Access: | Android SDK | Android SDK | dotNET SDK | Parse Server | Parse Server, Docs |
-| Focus:  | Android SDK | Android SDK | dotNET SDK | Parse Server | Parse Server |
+|                                                                                           | Name                                         | Role | Joined | Focus                         |
```
Comment (davimacedo) [inline TEAM.md:18]: Not sure if we should include focus for PMC (at lest at this moment)

**Review by davimacedo — COMMENTED** [review]
Could you please give me access to the branch so I can include the other team members? @TomWFox @dplewis any comment about this PR?

**Comment by mtrezza** [comment]
Sure, I've sent an invite.

File: TEAM.md (hunk 1 above)
Comment (mtrezza) [reply TEAM.md:3]: fixed

File: TEAM.md (hunk 2 above)
Comment (mtrezza) [reply TEAM.md]: removed

**Comment by davimacedo** [comment]
@mtrezza I've just completed the team section. Could you please take a look?

File: TEAM.md [hunk 3]
```diff
@@ -1,51 +1,66 @@
...
-|---------|:---:|:---:|:---:|:---:|:---:|
-| Name:   | [__John Carlson__](https://github.com/Jawnnypoo) | [__Roger Hu__](https://github.com/rogerhu) | [__Tobias__](https://github.com/TobiasPott) | [__Omair Vaiyani__](https://github.com/omairvaiyani) | [__Antoine Cormouls__](https://github.com/Moumouls) |
-| Access: | Android SDK | Android SDK | dotNET SDK | Parse Server | Parse Server, Docs |
-| Focus:  | Android SDK | Android SDK | dotNET SDK | Parse Server | Parse Server |
+|                                                                                           | Name                                         | Role | Joined | Focus |
+|-------------------------------------------------------------------------------------------|----------------------------------------------|------|--------|-------|
+| <img src="https://avatars.githubusercontent.com/u/4430059?v=4" width="80" height="80" />  | [Davi Macedo](https://github.com/davimacedo) | PMC  | -      | -     |
```
Comment (mtrezza) [inline TEAM.md:20]: Do you know the join dates of the PMC members?

Comment (davimacedo) [reply TEAM.md:20]: Maybe we can consider that the PMC actually started when this repo started. So it would be the date of the first commit.

Comment (mtrezza) [reply TEAM.md:20]: Sounds good, so everyone except me started when this repo started?

Comment (davimacedo) [reply TEAM.md:20]: yes

**Comment by mtrezza** [comment]
@davimacedo I have simplified this *a lot*, so we stop spending resources on this team page and finally merge this PR.

- Removed alumni
- Removed contributor list, there is a GH page for this; doesn't have to be updated manually, especially as we expect this to become more dynamic

**Review by davimacedo — APPROVED** [review]

**Pull request merged**